gapis/resolve/command_tree.go

```
@@ -340,21 +355,8 @@ func (r *CommandTreeResolvable) Resolve(ctx context.Context) (interface{}, error
 	return out, nil
 }
 
-func addDrawAndFrameEvents(ctx context.Context, p *path.CommandTree, t *commandTree, last api.CmdID) error {
-	events, err := Events(ctx, &path.Events{
-		Capture:      t.path.Capture,
-		Filter:       p.Filter,
-		DrawCalls:    p.GroupByDrawCall,
-		FirstInFrame: true,
-		LastInFrame:  true,
-	})
-	if err != nil {
-		return log.Errf(ctx, err, "Couldn't get events")
-	}
-
+func addDrawEvents(ctx context.Context, events *service.Events, p *path.CommandTree, t *commandTree, last api.CmdID) {
```

What's the fix for the "silently ignored error" of which you speak?